ws-nextjs-app/pages/[service]/live/[id]/Header/styles.tsx

```diff
@@ -5,6 +5,7 @@ export default {
   backgroundColor: ({ palette }: Theme) =>
     css({
       backgroundColor: palette.GREY_10,
+      borderBottom: `solid ${pixelsToRem(3)}rem transparent`,
```

Comment:
Should this be wrapped in `[mq.HIGH_CONTRAST]: {}` to ensure it only appears under that media query?